go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	// Do not bump testcontainers version. This will cause conflicts with go.opentelemetry.io because of a very old version
 	// found in steve. This is used just for testing.
```

Comment:
@weyfonk beware of this comment 😅 cc/ @raulcabello

Reply:
Argh! Thanks @aruiz14 :sweat_smile: 
Didn't we recently [remove](https://github.com/rancher/gitjob/commit/42e1f15558dd2b6f734da753f4a4668d4601fc53) steve dependencies though?

Reply:
Oh indeed! Nice timing! 😄